config/env.go

```diff
@@ -0,0 +1,35 @@
+package config
+
+// TODO: everything in this file should be either removed or unexported
+import (
+	"os"
+	"strconv"
+	"strings"
+)
+
+// ConfigKeyToEnvRudder as package method to get the formatted env from a given string
+func ConfigKeyToEnvRudder(s string) string {
```

Comment:
Why do we need `Rudder` in the name?

Reply:
we don't! To my defence, there used to be a `ConfigKeyToEnvViper` once, but decided it was not needed after all. Will rename